selfdrive/car/hyundai/values.py

```diff
@@ -235,6 +236,9 @@ def init_make(self, CP: car.CarParams):
     HyundaiCarInfo("Kia EV6 (without HDA II) 2022", "Highway Driving Assist", harness=Harness.hyundai_l),
     HyundaiCarInfo("Kia EV6 (with HDA II) 2022", "Highway Driving Assist II", harness=Harness.hyundai_p)
   ],
+  CAR.KIA_CARNIVAL_4TH_GEN: [
+    HyundaiCarInfo("Kia Carnival (China) 2023", harness=Harness.hyundai_k),
```

Comment:
Why China only?

Reply:
> Why China only?

The user mentioned that US and China models use different harnesses, and we currently don't have any confirmation from any users with the US model yet. We can always change it once we have confirmation.

Reply:
Could it be the same exception as what we have for the EV6 and Ioniq 5? "Southeast Asia only"?

Reply:
We have a U.S. version that is confirmed to use a different harness. Will have them separated in this PR.